src/components/link.js

```diff
@@ -28,6 +28,7 @@ module.exports.Component = registerComponent('link', {
     this.quaternionClone = new THREE.Quaternion();
     // Store hidden elements during peek mode so we can show them again later.
     this.hiddenEls = [];
+    this.previewDistance = 15;
```

Comment:
Add this as a constant at the top. Similar to https://github.com/aframevr/aframe/blob/master/src/components/gearvr-controls.js#L15

Reply:
fixed by commit 312efe07522cef3abc1b1f8a4f66dad6a1c9643f